testing-internal/src/main/java/com/linecorp/centraldogma/testing/internal/ExpectedExceptionAppender.java

```diff
@@ -0,0 +1,269 @@
+    }
+
+    @Override
+    protected void append(ILoggingEvent eventObject) {
+        final IThrowableProxy exception = getException(eventObject);
+            for (String exceptionName : exceptionAndCause.keySet()) {
+            if (exception != null && exception.getClassName().endsWith(exceptionName) &&
+                containsCause(exception, exceptionName)) {
+                aai.appendLoopOnAppenders(new ExpectedExceptionEventWrapper(eventObject));
+                return;
+            }
+        }
+        aai.appendLoopOnAppenders(eventObject);
+    }
+
+    private IThrowableProxy getException(ILoggingEvent event) {
+        final IThrowableProxy throwableProxy = event.getThrowableProxy();
+        if (throwableProxy != null) {
+            final IThrowableProxy cause = throwableProxy.getCause();
+            if (cause != null) {
+                return cause;
+            }
+        }
+        return null;
+    }
+
+    private boolean containsCause(IThrowableProxy exception, String exceptionName) {
+        final String message = exception.getMessage();
+        if (message != null && message.contains(exceptionAndCause.get(exceptionName))) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public void start() {
+        if (!aai.iteratorForAppenders().hasNext()) {
+            addWarn("No appender was attached to " + getClass().getSimpleName() + '.');
+        }
+        super.start();
+    }
+
+    @Override
+    public void stop() {
+        try {
+            aai.detachAndStopAllAppenders();
+        } finally {
+            super.stop();
+        }
+    }
+
+    @Override
+    public void addAppender(Appender<ILoggingEvent> newAppender) {
+        aai.addAppender(newAppender);
+    }
+
+    @Override
+    public Iterator<Appender<ILoggingEvent>> iteratorForAppenders() {
+        return aai.iteratorForAppenders();
+    }
+
+    @Override
+    public Appender<ILoggingEvent> getAppender(String name) {
+        return aai.getAppender(name);
+    }
+
+    @Override
+    public boolean isAttached(Appender<ILoggingEvent> appender) {
+        return aai.isAttached(appender);
+    }
+
+    @Override
+    public void detachAndStopAllAppenders() {
+        aai.detachAndStopAllAppenders();
+    }
+
+    @Override
+    public boolean detachAppender(Appender<ILoggingEvent> appender) {
+        return aai.detachAppender(appender);
+    }
+
+    @Override
+    public boolean detachAppender(String name) {
+        return aai.detachAppender(name);
+    }
+
+    private static final class ExpectedExceptionEventWrapper implements ILoggingEvent {
+        private final ILoggingEvent event;
+
+        ExpectedExceptionEventWrapper(ILoggingEvent event) {
+            this.event = event;
+        }
+
+        @Override
+        public Object[] getArgumentArray() {
+            return event.getArgumentArray();
+        }
+
+        @Override
+        public Level getLevel() {
+            return Level.DEBUG;
```

Comment:
How about returning `event.getLevel()` if it's lower than `Level.DEBUG`?

Reply:
Changed it~!!